dEdX/steer/steer_example.xml

```diff
@@ -35,7 +12,8 @@
     <constant name="productionfolder" value="/cvmfs/ilc.desy.de/sw/ILDConfig/v02-00-01/StandardConfig/production" />
```

Comment:
```suggestion
    <constant name="productionfolder" value="/cvmfs/ilc.desy.de/sw/ILDConfig/v02-03/StandardConfig/production" />
```
Probably needs to be adjusted in any case, but at least point to a more recent version of ILDConfig.